lib/kaitai/console_windows.rb

```diff
@@ -1,266 +1,262 @@
-    draw_rectangle(10, top_y, @cols - 20, 10)
-    goto(@cols / 2 - (header.length / 2) - 1, top_y)
-    print ' ', header, ' '
-    goto(11, top_y + 1)
-    puts msg
-    draw_button(@cols / 2 - 10, top_y + 8, 10, 'OK')
-    loop {
-      c = read_char_mapped
-      return if c == :enter
-    }
-  end
+    SINGLE_CHARSET = '┌┐└┘─│'
+    HEAVY_CHARSET  = '┏┓┗┛━┃'
+    DOUBLE_CHARSET = '╔╗╚╝═║'
+
+    CHAR_TL = 0
+    CHAR_TR = 1
+    CHAR_BL = 2
+    CHAR_BR = 3
+    CHAR_H  = 4
+    CHAR_V  = 5
+
+    def message_box(header, msg)
+      top_y = @rows / 2 - 5
+      draw_rectangle(10, top_y, @cols - 20, 10)
+      goto(@cols / 2 - (header.length / 2) - 1, top_y)
+      print ' ', header, ' '
+      goto(11, top_y + 1)
+      puts msg
+      draw_button(@cols / 2 - 10, top_y + 8, 10, 'OK')
+      loop do
+        c = read_char_mapped
+        return if c == :enter
+      end
+    end
 
-  def input_str(header, msg)
-    top_y = @rows / 2 - 5
-    draw_rectangle(10, top_y, @cols - 20, 10)
-    goto(@cols / 2 - (header.length / 2) - 1, top_y)
-    print ' ', header, ' '
+    def input_str(header, _msg)
+      top_y = @rows / 2 - 5
+      draw_rectangle(10, top_y, @cols - 20, 10)
+      goto(@cols / 2 - (header.length / 2) - 1, top_y)
+      print ' ', header, ' '
 
-    goto(11, top_y + 1)
-    Readline.readline('', false)
-  end
+      goto(11, top_y + 1)
+      Readline.readline('', false)
+    end
 
-  def draw_rectangle(x, y, w, h, charset = DOUBLE_CHARSET)
-    goto(x, y)
-    print charset[CHAR_TL]
-    print charset[CHAR_H] * (w - 2)
-    print charset[CHAR_TR]
-
-    ((y + 1)..(y + h - 1)).each { |i|
-      goto(x, i)
-      print charset[CHAR_V]
-      print ' ' * (w - 2)
-      print charset[CHAR_V]
-    }
-
-    goto(x, y + h)
-    print charset[CHAR_BL]
-    print charset[CHAR_H] * (w - 2)
-    print charset[CHAR_BR]
-  end
+    def draw_rectangle(x, y, w, h, charset = DOUBLE_CHARSET)
+      goto(x, y)
+      print charset[CHAR_TL]
+      print charset[CHAR_H] * (w - 2)
+      print charset[CHAR_TR]
+
+      ((y + 1)..(y + h - 1)).each do |i|
+        goto(x, i)
+        print charset[CHAR_V]
+        print ' ' * (w - 2)
+        print charset[CHAR_V]
+      end
+
+      goto(x, y + h)
+      print charset[CHAR_BL]
+      print charset[CHAR_H] * (w - 2)
+      print charset[CHAR_BR]
+    end
 
-  def draw_button(x, y, w, caption)
-    goto(x, y)
-    puts "[ #{caption} ]"
-  end
+    def draw_button(x, y, _w, caption)
+      goto(x, y)
+      puts "[ #{caption} ]"
+    end
 
-  # Regexp borrowed from
-  # http://stackoverflow.com/questions/170956/how-can-i-find-which-operating-system-my-ruby-program-is-running-on
-  @@is_windows = (RUBY_PLATFORM =~ /cygwin|mswin|mingw|bccwin|wince|emx/) ? true : false
+    # Regexp borrowed from
+    # http://stackoverflow.com/questions/170956/how-can-i-find-which-operating-system-my-ruby-program-is-running-on
+    @@is_windows = RUBY_PLATFORM =~ /cygwin|mswin|mingw|bccwin|wince|emx/ ? true : false
```

Comment:
Brackets should be kept in my  opinion, is more readable with them.

Reply:
Incorporated this change with https://github.com/kaitai-io/kaitai_struct_visualizer/pull/37/commits/049dadb6fc63330387c05c68aac4d3917896dd17.